data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	"Do not shout victory too soon, however, for we were not the only ones you had to beat," they add.`
+			label "good boy or bad boy enough"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"old reputation: Hai (Unfettered)" < -2000
+			has "flagship model: Ancient Ladybug"
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained entering plunders staying target heroic
+		ship "Sea Scorpion (Shredder)" "Requiem"
+		system "Hi Yahr"
+		dialog "You have managed to defeat the last of the Unfettered challengers, it is now time to land to claim your reward."
+		on disable
+			set "unfettered: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		ship "Lightning Bug (Unfettered Shipyards)"
+		personality launching staying surveillance
+		on disable
+			dialog `The engineer hails you, "What do you think you are doing? The duel will not be valid if you destroy my ship."`
+		on kill
+			fail "Hai Legacy 3"
+	on visit
+		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight. Load an earlier save and try again if you want to finish this mission. You may also have capped too many of their ships, in which case you should disown them."
+	to complete
+		has "unfettered: beaten the bugs"
+		has "flagship model: Ancient Ladybug"
+		# If they bended the rules you can get away with capping one ship.
+		"total ships" <= 1 + "unfettered: beaten extra bugs"
+		or
+			"old reputation: Hai (Unfettered)" > -2000
+			has "unfettered: beaten extra bugs"
+	on complete
+		conversation
+			`The engineer seems shocked at first. "You really... won? It really was close until the end, was it not? Or did the captains already make stories to justify their incompetence?"`
+			choice
+				`	"Indeed, but it was fun."`
+					goto next
+				`	"What are you talking about? This was easy!"`
+			`	"Don't become overconfident now, Captain. I am sure you know what happens to such people.`
+			label next
+			`	"I must congratulate you, I never expected you had it in you. I only have one word and will give you authorization to land, on the condition that you keep this relic from our past in good condition!`
+			`	"Make sure you do not lose it until you truly earn the right to call us friends, or we will take it very seriously. Oh, and there are taxes on the daily fee, totaling 681 credits per day. I hope you don't mind."`
+			choice
+				`	"Thank you for your trust."`
+					goto trust
+				`	"I refuse to pay much that for simple access."`
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				log "Won a fight against a few Lightning Bugs in an Ancient Ladybug, but decided against paying the fee of 681 to land."
+			`	"What? Oh, well the winner of the challenge can always refuse the spoils, I suppose. In that case our conflict will resume."`
```

Comment:
Can the player refuse the negative aspect of having to pay without losing the other benefits (if there's a reputation change)?

Reply:
isnt that a bit too easy? you get the positive but not the negatives?
the license is the balanced way to give you access to uhai space

Reply:
Got it. Probably just add to your conversation somewhere that clarifies this is an "all or nothing" type deal for the Unfettered. Either you pay the fee and get positive relations, or you don't pay the fee and the Unfettered won't treat you any differently.

Reply:
it's a landing permit after all but got it, added